test-image.bats

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env bats
 
 @test "Test for node and version" {
-  run docker run --rm -it node:"$full_tag" node -e "process.stdout.write(process.versions.node)"
```

Comment:
@nschonni would it work if you only removed the psuedo-tty `-t` but left the `-i`?  If it's in CI, it shouldn't need the tty anyways right?  

`docker run --rm -i node:"$full_tag"`

Reply:
Not sure, but these tests also just overlap what is done in the image build now, so i'm not really concerned

Reply:
yeah, was just curious since i'll soon have to test the same thing, and you already have everything set up :rofl: :thumbsup: